### ISSUE TITLE
bug: transition only covers 16:9 aspect ratios

### ISSUE THREAD

**Issue by willuhmjs** [LeoDog896/rotator#1]
The transition between levels only covers the entire screen on 16:9 aspect ratios. Otherwise, it only covers part of the screen.

![image](https://user-images.githubusercontent.com/58738637/222916022-8811dd25-fab1-4ce3-9752-56dcd7a3f83f.png)

**Comment by tristan-f-r** [comment]
Yeah, I noticed this while porting it - it seems to be an issue with how the ColorRect is layered on the player - it should be possible to anchor it in the scene.

**Comment by tristan-f-r** [comment]
Actually, horrors! The UI node is _attatched_ to the player for some odd reason.
